app/main.py

```diff
@@ -1,3 +1,59 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> int:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> int:
```

Comment:
Change your type annotation to indicate that you return a string, not an integer.